erpnext/crm/report/lead_details/test_lead_details.py

```
@@ -0,0 +1,100 @@
+		self.assertTrue(len(columns) > 0, "Report should return columns")
+		self.assertTrue(len(data) > 0, "Report should return at least one row")
+
+	def test_lead_details_columns_include_designation(self):
+		"""Test that the designation column is present in the report output."""
+		filters = frappe._dict(
+			company=self.company,
+			from_date=add_days(getdate(), -1),
+			to_date=add_days(getdate(), 1),
+		)
+
+		columns, data = execute(filters)
```

Ruff (0.15.12)

[warning] Unpacked variable `data` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)